source/getting_started/aiot_intelligent_edge/coco_for_devices/linux_coco_for_devices/quick_start_guide_linux_coco_for_devices.rst

```diff
@@ -3,6 +3,158 @@
 Quick Start Guide
 =================
 
-..
-    <Content here - Owner: Theja
-    Suggestions from Ashish - Steps to Sync Sources, Compilation, and Running Successfully. Also mentioning how Auto-Magically things connect to the COCO App>
+Build and run your first Device App in a few short steps by following
+the guide below.
+
+You will need: 
+
+1. Ubuntu to run the app 
+2. gcc to compile it.
+
+1. ‘Signup’ for a new COCO account or ‘Sign In’ if you have one by following this `link here <https://manage.getcoco.buzz/>`__
+------------------------------------------------------------------------------------------------------------------------------
+
+.. image:: quick_start_guide_linux_coco_for_devices/step1.png
+   :alt: Step 1
+
+2. Create a Product for running the Sample Device App (usually will be a Product representing the SKU of the devices to be deployed).
+-------------------------------------------------------------------------------------------------------------------------------------
+
+a. Click on 'Devices’
+
+.. image:: quick_start_guide_linux_coco_for_devices/step2a.png
+   :alt: Step 2a
+
+b. Click on ‘(+) Product’
+
+.. image:: quick_start_guide_linux_coco_for_devices/step2b.png
+   :alt: Step 2b
+
+c. Fill in the details for a new Product.
+
+.. image:: quick_start_guide_linux_coco_for_devices/step2c.png
+   :alt: Step 2c
+
+3. Create a license for a device, in order to run the Sample Device App
+-----------------------------------------------------------------------
+
+a. Click on your Product to go to the Product Details page.
+
+.. image:: quick_start_guide_linux_coco_for_devices/step3a.png
+   :alt: Step 3a
+
+b. Click on ‘Device Licenses’
+
+.. image:: quick_start_guide_linux_coco_for_devices/step3b.png
+   :alt: Step 3b
+
+c. Enter the number of licenses required as 1 (usually will be entered
+   as the no. of devices to be deployed for the Product).
+
+.. image:: quick_start_guide_linux_coco_for_devices/step3c.png
+   :alt: Step 3c
+
+4. Set up the Sample Device App
+-------------------------------
+
+a. Download the license file for a device, using the ‘Download’ icon.
+
+.. image:: quick_start_guide_linux_coco_for_devices/step4a.png
+   :alt: Step 4a
+
+b. Download cocodevicesdk.so
+
+c. Download the Sample Device App source code and
+   compile it by linking it to cocodevicesdk.so
+
+d. Copy the downloaded license file to the host where you intend to run
+   the Sample Device App, and rename it to **config.txt**
+
+5. Run the Sample Device App.
+-----------------------------
+
+a. Run the Sample Device App using a command similar to:
+
+   ::
+
+       sudo <Sample Device App path>
```

Comment:
Change this to the command line I had given with placeholders for necessary parameters